docs/troubleshooting/firebase-issues/firebase_storage_limits_in_flutterflow.md

```diff
@@ -0,0 +1,49 @@
+---
+keywords: ['storage', 'firebase', 'flutterflow']
+slug: /firebase-storage-limits-in-flutterflow
+title: Firebase Storage Limits in FlutterFlow
+---
+
+# Firebase Storage Limits in FlutterFlow
+
+Managing Firebase Storage properly is essential for controlling your app's file storage and associated costs in FlutterFlow. This article summarizes the key limits and best practices.
+
+## Firebase Storage Plans and Limits
+
+- **Spark Plan (Free)**
```

Comment:
Please refer to the latest info from Firebase docs, because as far as I remember, Firebase Storage isnt free anymore?

Reply:
can you point to the docs that says its 1 GB?